### PR TITLE
feat: add devcard page

### PR DESCRIPTION
**Pull request by kopancek**
Screenshot 2024-02-08 at 19 03 45

Screenshot 2024-02-12 at 12 36 24

### CODE REVIEW

**Review by sshanzel — APPROVED** [review]
Looks great! 🚀

**Review by rebelchris — CHANGES_REQUESTED** [review]

File: packages/webapp/pages/devcards/[userId].tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+export async function getStaticProps({
+  params,
+}: GetStaticPropsContext): Promise<GetStaticPropsResult<DevCardPageProps>> {
+  const { userId } = params;
+
+  if (!userId) {
+    return {
+      notFound: true,
+      revalidate: 60,
```
Comment (capJavert) [inline packages/webapp/pages/devcards/[userId].tsx:28]: we can probably set revalidate to something more like a day or`false` so its always fresh whey making screenshot.

**Review by capJavert — APPROVED** [review]

File: packages/webapp/pages/devcards/[userId].tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  return {
+    props: { userId: userId as string },
+    revalidate: 60,
+  };
+}
+
+const DevCardPage = ({ userId }: DevCardPageProps): ReactElement => {
+  return (
+    <div id="screenshot_wrapper" className="w-[356px]">
```
[the remark this answers is not in the document]
Comment (sshanzel) [reply packages/webapp/pages/devcards/[userId].tsx:40]: Would this not produce extra spaces on download? I think our devcard in itself is only 324px. We probably don't need the className?

File: packages/webapp/pages/devcards/[userId].tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      <DevCard userId={userId} type={DevCardType.Vertical} />
```
Comment (sshanzel) [inline packages/webapp/pages/devcards/[userId].tsx:41]: I think the API introduces a type property as part of the query params. Should we utilize it here and have the `DevCardType.Vertical` as fallback?

File: packages/webapp/pages/devcards/[userId].tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (sshanzel) [reply packages/webapp/pages/devcards/[userId].tsx:40]: Might also become an issue when the type is `horizontal`.

File: packages/webapp/pages/devcards/[userId].tsx (hunk 3 above)
Comment (kopancek) [reply packages/webapp/pages/devcards/[userId].tsx:41]: Right now we only support the vertical type. The API is not sending query params at the moment. That's an improvement waiting to happen :)

Comment (kopancek) [reply packages/webapp/pages/devcards/[userId].tsx]: Fixed 👍

File: packages/webapp/pages/devcards/[userId].tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (kopancek) [reply packages/webapp/pages/devcards/[userId].tsx]: This was just a placeholder. Replaced with `w-fit` instead.

**Review by sshanzel — APPROVED** [review]
Looks amazing! I will just have to update my PR for twitter cover to reflect the correct type in the map here.

**Pull request merged**